src/experiment/trials/holdUpMarker.js

```diff
@@ -1,32 +1,12 @@
 import htmlButtonResponse from "@jspsych/plugin-html-button-response";
+import { LANGUAGE } from "../../config/main";
+import { h1 } from "../../lib/markup/tags";
+import { photodiodeGhostBox } from "../../lib/markup/photodiode";
 
-import { config, LANGUAGE } from "../../config/main";
-import { eventCodes } from "../../config/trigger";
-import { pdSpotEncode, photodiodeGhostBox } from "../../lib/markup/photodiode";
-import { div, h1, p } from "../../lib/markup/tags";
-
-// TODO @brown-ccv #330: Custom extension for EEG - this is initializeTriggerBox
-// TODO @brown-ccv #330: Need to ping the serial part - this isn't doing anything yet
-// TODO @brown-ccv: Prevent user from pressing continue until pdSpotEncode finishes (need to use jsPsych.pluginAPI.setTimeout)
 export const holdUpMarkerTrial = {
```

Comment:
Just adding back some of the TODOs that got deleted
```suggestion
// TODO @brown-ccv #330: Custom extension for EEG - this is initializeTriggerBox
// TODO @brown-ccv #330: Need to ping the serial part - this isn't doing anything yet
export const holdUpMarkerTrial = {
```